### DISCUSSION TITLE
Do not get option to import when I click on the 'add node' option in Windows.

### DISCUSSION THREAD

**Discussion by NO00636712** [cyberbotics/webots#5295] [Q&A]
I was following a tutorial on running Deepbots for reinforcement Learning on Webots. I downloaded a .wbo file and when I tried to import using the 'add' button following the tutorial, I noticed I did not have the option to import. I am working on a Windows 10 machine.

**Comment by omichel** [comment]
The `.wbo` format is not supported any more in Webots R2022b (see [change log](https://cyberbotics.com/doc/reference/changelog-r2022#webots-r2022b)). You should convert this file into a `.proto` file in order to use it with Webots R2022b. The conversion can easily be achieved with a simple text editor, following the [proto syntax](https://www.cyberbotics.com/doc/reference/proto).
